src/dataset.jl

```diff
@@ -713,7 +713,9 @@ function getgeotransform!(
 )::Vector{Cdouble}
     @assert length(transform) == 6
     result = GDAL.gdalgetgeotransform(dataset.ptr, pointer(transform))
-    @cplerr result "Failed to get geotransform"
+    if result != GDAL.CE_None
+        transform .= (0,1,0,0,0,1)
```

Comment:
Nitpick, making these floats is more explicit.

Reply:
Done in https://github.com/yeesian/ArchGDAL.jl/pull/236/commits/71a01e8682cd03e434f5eb7a237c976babd95c7d.